.env.example

```diff
@@ -6,7 +6,7 @@ APP_URL=http://localhost
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
-DB_DATABASE=homestead
+DB_DATABASE=ringme
```

Comment:
I should use homestead. Because the moest developers use homestead to try laravel projects.